sources.md

```diff
@@ -0,0 +1,82 @@
+# Data Sources
+
+This project uses data from various sources that are openly licensed or in the public domain. Below are the sources and their respective information:
+
+## CC Legal Tools
+
+**Description:** A .txt provided by Timid Robot containing all legal tool paths. The data from Google Custom Search will only cover 50+ general, most significant categories of CC License for data collection quota constraint. As an additional note, the order of precedence of license the collected data's first column is sorted due to intermediate data analysis progress.
+
+**API documentation link:** [List of all current CC legal tool paths by TimidRobot](https://github.com/creativecommons/quantifying/blob/main/google_custom_search/legal-tool-paths.txt)
+
+**API information:**
+- No API key required
+- No query limits
+
+## Flickr
+
+**Description:** The Flickr API exposes identifiers for users, photos, photosets and other uniquely identifiable objects.
+
+**API documentation link:** [API documentation - Flickr Services](https://www.flickr.com/services/api/)
+
+**API information:**
+- API key required
+- Query limit: 3600 requests per hour
+- Data available through CSV format
+
+## Google Custom Search JSON API
+
+**Description:** The Custom Search JSON API allows user-defined detailed query and access towards related query data using a programmable search engine.
+
+**API documentation links:**
+- [Custom Search JSON API Reference | Programmable Search Engine | Google Developers](https://developers.google.com/custom-search/v1/reference/rest)
+- [Method: cse.list | Custom Search JSON API | Google Developers](https://developers.google.com/custom-search/v1/reference/rest/v1/cse/list)
+
+**API information:**
+- API key required
+- Query limit: 100 queries per day for free version
+- Data available through JSON format
+
+## Internet Archive Python Interface
+
+**Description:** A python interface to archive.org to achieve API requests towards internet archive.
+
+**API documentation link:** [internetarchive.Search - Internetarchive: A Python Interface to archive.org](https://internetarchive.readthedocs.io/en/stable/internetarchive.html#internetarchive.Search)
+
+**API information:**
+- No API key required
+- No query limits
+
+## The Metropolitan Museum of Art Collection API
+
+- Description: An API endpoint for receiving Metropolitan Muesum of Art Collection's CC-Licensed works.
+- API documentation link: [Latest Updates | The Metropolitan Museum of Art Collection API](https://metmuseum.github.io/)
+- API information:
+  - No API key required
+  - No query limits
```

Comment:
The query limit is 80 requests per second. 
<img width="501" alt="image" src="https://user-images.githubusercontent.com/97350598/222767832-75906483-4a4f-48dc-b9d3-a6f4941a11ec.png">

Reply:
Hey bro, I appreciate your effort in pointing out the issues you found in my pr. However, instead of just pointing out the mistake, why don't we collaborate and work together to fix it? We can submit a pull request together, which would make it easier for the mentor of this repo to review and merge. So, why not send a pull request to my repo and let's work on this together?

Reply:
Hey, sure! Working on it seems like a better idea. I was just asked to review the PR hence was suggesting required changes. It's my first time contributing so I'm sorry if this was the wrong way to approach things!

Reply:
Are you available on slack? Perhaps we can converse there for better feasibility.

Reply:
Hi PurviChaurasia,

Thank you for your message. Yes, I am available on Slack. My Slack ID is U04S3CACC2J and I'll be happy to converse with you there.

I apologize if my previous message sounded rude or offensive in any way. I appreciate your effort to contribute to the project and I agree that working on it together would be a great idea. Let's discuss the feasibility on Slack and work on it collaboratively.

Looking forward to hearing from you on Slack.

Best regards,

Reply:
Great thanks, will ping you there :)

Reply:
Thanks for your suggestion, I completely agree!

Reply:
Great, I actually went through the rest of the APIs and all of them seem in place hence there isn't a need to collaborate on this PR any further. However, I am always open to the opportunity!

Reply:
Thank you @PurviChaurasia  for pointing out the typos. Your help is much appreciated. I wish you the best of luck for your future endeavors.